src/nanoemoji/nanoemoji.py

```diff
@@ -83,11 +83,33 @@ class ColorGenerator(NamedTuple):
 
 
 _COLOR_FORMAT_GENERATORS = {
+<<<<<<< Updated upstream
     "glyf": ColorGenerator(lambda *args: _glyf_ufo(*args), lambda *_: None),
     "colr_0": ColorGenerator(lambda *args: _colr_ufo(0, *args), lambda *_: None),
     "colr_1": ColorGenerator(lambda *args: _colr_ufo(1, *args), lambda *_: None),
     "svg": ColorGenerator(lambda *_: None, lambda *args: _svg_ttfont(*args, zip=False)),
     "svgz": ColorGenerator(lambda *_: None, lambda *args: _svg_ttfont(*args, zip=True)),
+=======
+    "glyf": ColorGenerator(lambda *args: _glyf_ufo(*args), lambda *_: None, ".ttf"),
+    "glyf_colr_0": ColorGenerator(
+        lambda *args: _colr_ufo(0, *args), lambda *_: None, ".ttf"
+    ),
+    "glyf_colr_1": ColorGenerator(
+        lambda *args: _colr_ufo(1, *args), lambda *_: None, ".ttf"
+    ),
+    "cff_colr_0": ColorGenerator(
+        lambda *args: _colr_ufo(0, *args), lambda *_: None, ".otf"
+    ),
+    "cff_colr_1": ColorGenerator(
+        lambda *args: _colr_ufo(1, *args), lambda *_: None, ".otf"
+    ),
+    "svg": ColorGenerator(
+        lambda *_: None, lambda *args: _svg_ttfont(*args, compressed=False), ".ttf"
+    ),
+    "svgz": ColorGenerator(
+        lambda *_: None, lambda *args: _svg_ttfont(*args, compressed=True), ".ttf"
+    ),
+>>>>>>> Stashed changes
```

Comment:
And here :)

Reply:
Heh I thought I'd have time to fix that before you looked!